Controller/Cells/LeaderboardTableViewCell.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  TableViewCell.swift
+//  TheCoachTimer
+//
+//  Created by Pierluigi Guardascione on 23/11/21.
+//
+
+import UIKit
+
+class LeaderboardTableViewCell: UITableViewCell {
+    
+    @IBOutlet weak var playerImage: UIImageView!
+    
+    @IBOutlet weak var playerNameLabel: UILabel!
+    
+    @IBOutlet weak var maxVelocityValue: UILabel!
+    
+    @IBOutlet weak var lapsNumberValue: UILabel!
+    
+    
+    
+
+    override func awakeFromNib() {
```

Comment:
```suggestion
    @IBOutlet weak var playerImage: UIImageView!
    @IBOutlet weak var playerNameLabel: UILabel!
    @IBOutlet weak var maxVelocityValue: UILabel!
    @IBOutlet weak var lapsNumberValue: UILabel!
    
    override func awakeFromNib() {
```

In general I see a lot of white space, that is good when grouping maybe variables, but would be better to be compact.